temporalio/ui
# remove feature guard for configurable workflows table

## Description & motivation 💭 
This removes the prop from `<WorkflowsWithNewSearch />` that controls whether the configurable table should be rendered. With the release of this feature to cloud, the old table will be unused, so I also deleted that.
### Screenshots (if applicable) 📸 
N/a
### Design Considerations 🎨 
N/a
## Testing 🧪 

### How was this tested 👻 

### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
- run core locally - ensure configurable table is enabled by default
- run cloud locally with local version of core package linked - ensure configurable table is enabled by default

## Checklists
N/a
### Draft Checklist 

### Merge Checklist 

### Issue(s) closed 
N/a
## Docs
N/a
### Any docs updates needed?
